packages/modal/src/stories/ModalCustom.bs.stories.ts

```diff
@@ -27,34 +38,35 @@ export default {
   },
 }
 
-type StoryModel =
-  | UiModalModel
-  | {
-      titleSlot: string
-      bodySlot: string
-      footerSlot: string
-    }
+type StoryModel = UiModalModel & {
+  titleSlot: string
+  bodySlot: string
+  footerSlot: string
+}
 
 const Template = (args: StoryModel) => ({
-  components: { UiModal },
+  components: { UiModal, UiCodeHighlight },
```

Comment:
Might I would be wrong, but you need to add code highlight in the template, don't you?

Reply:
Done